nameko/events.py

```diff
@@ -197,19 +241,24 @@ def setup(self):
             queue_name = "evt-{}-{}".format(self.source_service,
                                             self.event_type)
         elif self.handler_type is BROADCAST:
+            broadcast_identifier = self.broadcast_identifier
             queue_name = "evt-{}-{}--{}.{}-{}".format(self.source_service,
                                                       self.event_type,
                                                       service_name,
                                                       self.method_name,
-                                                      uuid.uuid4().hex)
+                                                      broadcast_identifier)
 
         exchange = get_event_exchange(self.source_service)
 
         # auto-delete queues if events are not reliably delivered
         auto_delete = not self.reliable_delivery
+
+        # broadcast queues are exclusive
+        exclusive = self.handler_type is BROADCAST
```

Comment:
Ah. It turns out that `exclusive` implies `auto_delete` even if you request otherwise (and nothing complains if you supply both). I will change it so we only use `exclusive` when reliable delivery is off.